src/app/garfile/submit/success/index.njk

```diff
@@ -18,50 +18,34 @@
 
 {% block content %}
 
-
   {% import "common/templates/includes/macros.njk" as m %}
 
   {% if errors %}
     {% include "common/templates/includes/errors.njk" %}
   {% endif %}
 
   <div class="govuk-grid-row">
-      <div class="govuk-panel govuk-panel--confirmation">
-        <h1 class="govuk-panel__title">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
           GAR submitted
-        </h1>
-        <div class="govuk-panel__body">
-          Your reference number is
-        <br><strong>{{cookie.getGarId()}}</strong>
-        </div>
-      </div>
-
-       <p class="govuk-body-m">
-         We have sent a confirmation email to {{cookie.getUserEmail()}}
-       </p>
-
-       <div class="govuk-warning-text">
-        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-        <strong class="govuk-warning-text__text">
-          <span class="govuk-warning-text__assistive">{{__('assistive_warning')}}</span> 
-          New: when the craft departs, you need to let Border Force know if anyone in the GAR did not not travel with it.<br />
-          This can be done from the <a href="/home">Manage GAR</a> page in the 'Departure' tab.
-        </strong>
-      </div>
+      </h1>
 
-       <h2 class="govuk-heading-m">
-         What happens next
-       </h2>
-
-       <p class="govuk-body-m">
+      <p class="govuk-body-m">
```

Comment:
Minor: From [FAIR 437](https://collaboration.homeoffice.gov.uk/jira/browse/FAIR-437), the text has poor contrast with dark blue-green color. Could be an accessibility problem or not?

Reply:
This passes WCAG AA so I think OK for the moment.